File: Auto-generated-tasks/Difficulty-1/Task-3/Solution.cs
int u, A2, Num = 6;
int[] Mas = new int[Num];
for (i = 0; i < Num; i++) {
	Random rnd = new Random();
	Mas[u] = rnd.Next(4, 422);
	A2 += Mas[u];
}
Console.WriteLine("A2=" + A2);

File: Auto-generated-tasks/Difficulty-1/Task-3/Solution.py
from random import *
u = 0
A2 = 0
Num = 6
Mas = [0] * Num
for u in range(0, Num):
	Mas[u] = randint(4, 422)
	A2 += Mas[u]
print("A2=", A2)